kubernetes-tests/src/test/java/io/fabric8/kubernetes/client/mock/CustomResourceCrudWithCRDContextTest.java

```diff
@@ -20,36 +20,32 @@
 import io.fabric8.kubernetes.client.KubernetesClient;
 import io.fabric8.kubernetes.client.dsl.MixedOperation;
 import io.fabric8.kubernetes.client.dsl.Resource;
-import io.fabric8.kubernetes.client.dsl.base.CustomResourceDefinitionContext;
 import io.fabric8.kubernetes.client.mock.crd.EntandoBundleRelease;
 import io.fabric8.kubernetes.client.mock.crd.EntandoBundleReleaseList;
 import io.fabric8.kubernetes.client.mock.crd.EntandoBundleReleaseSpec;
-import io.fabric8.kubernetes.client.server.mock.KubernetesServer;
+import io.fabric8.kubernetes.client.server.mock.EnableKubernetesMockClient;
 import io.fabric8.kubernetes.internal.KubernetesDeserializer;
-import org.junit.Rule;
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.migrationsupport.rules.EnableRuleMigrationSupport;
-
-import java.util.Collections;
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
 
-@EnableRuleMigrationSupport
+@EnableKubernetesMockClient(crud = true)
 class CustomResourceCrudWithCRDContextTest {
-  @Rule
-  public KubernetesServer kubernetesServer = new KubernetesServer(true, true, Collections.singletonList(new CustomResourceDefinitionContext.Builder()
-    .withScope("Namespaced")
-    .withPlural("entandobundlereleases")
-    .withVersion("v1alpha1")
-    .withGroup("demo.fabric8.io")
-    .withKind("EntandoBundleRelease")
-    .build()));
+//  @Rule
+//  public KubernetesServer kubernetesServer = new KubernetesServer(true, true, Collections.singletonList(new CustomResourceDefinitionContext.Builder()
+//    .withScope("Namespaced")
+//    .withPlural("entandobundlereleases")
+//    .withVersion("v1alpha1")
+//    .withGroup("demo.fabric8.io")
+//    .withKind("EntandoBundleRelease")
+//    .build()));
```

Comment:
Looks like providing CustomResourceDefinitionContext is no longer relevant since it can be configured from annotations in Pojos. Let's remove this commented out code.

Reply:
Yes, removed it!